### PR TITLE
Intel NUC desktop rebrand

### PR DESCRIPTION
**Pull request by carkod**
## QA

- Check https://ubuntu-com-13570.demos.haus/download/intel-nuc-desktop against prod/[copy docs](https://docs.google.com/document/d/1hDkrBqpJXWaENYZMSxd8P-XLFQ2LQqS1wNe4o2MGLho/edit#heading=h.r96oqmuwezfa)
- [Visual design] Check against screenshot below (it's a template, to be used as reference)

## Issue / Card

Fixes https://ubuntu-com-13570.demos.haus/

## Screenshots

managed

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by juanruitina** [comment]
The sections under "Next steps" don't exist on the live version, so please remove. I've adjusted the copy doc accordingly.

The form and the footer are missing, can you please incorporate them before I do a full review? Thank you.

**Comment by lyubomir-popov** [comment]
- wrong style here - please use an `hr.p-rule` above this, and make it an `.p-section>h2` followed by the image 

image

- pls strip the background from this, see the design: 
image


(review not finished, will continue tomorrow)

**Comment by carkod** [comment]
> * wrong style here - please use an `hr.p-rule` above this, and make it an `.p-section>h2` followed by the image
> 
> image
> 
> * pls strip the background from this, see the design:
>   image
> 
> (review not finished, will continue tomorrow)

Addressed this comment and the ones in MM

**Comment by lyubomir-popov** [comment]
Hi,

- as I mentioned on mattermost, the lines missing above list items on smaller breakpoints:
image

- stepped lists have 0 in the counter - no need for that:
image


- pls take this from MariaPaula's pr - this doesn't look like in the design:
 
image


and finally, might be worth using the same components - row--50-50 for example for the first section.


I'm pushing a fix - the first section should have `.p-strip is-shallow.u-no-padding--bottom>.p-section`; next release of vanilla will have p-section--hero that does all that with one class

**Comment by carkod** [comment]
> Hi,
> 
> * as I mentioned on mattermost, the lines missing above list items on smaller breakpoints:
>   image
> * stepped lists have 0 in the counter - no need for that:
>   image
> * pls take this from MariaPaula's pr - this doesn't look like in the design:
> 
> image
> 
> and finally, might be worth using the same components - row--50-50 for example for the first section.
> 
> I'm pushing a fix - the first section should have `.p-strip is-shallow.u-no-padding--bottom>.p-section`; next release of vanilla will have p-section--hero that does all that with one class

Done. The 0 counter is coming from [vanilla lists](https://vanillaframework.io/docs/patterns/lists), could be a bug?

**Comment by juanruitina** [comment]
- [ ] List under "3. Install Ubuntu Desktop" isn't numbered as it is now in the live version.
- [ ] Across all pages: the numeration in lists inside steps start by 0.
- [ ] Across all pages: the HR and the text inside steps don't expand to the right margin on phones (heading does), see below. Might have to do something with grid and gaps.

ubuntu-com-13580 demos haus_download_intel-nuc(iPhone SE)

Screenshot 2024-02-16 at 15 45 34

**Comment by lyubomir-popov** [comment]
- please remove the `<small>` - all elements in the form  form should use the same font-size as in the design
- the button is too close to the button, probably due to the markup  being split into list items. Inputs chould push down other content by ~1.5rem, as in the vanilla examples - 
image

I would also wrap it in a p-section, so the button is a bit further apart

- strange bullet (square) here:
image

decimal point bullet numbers, already flagged once - please check all new prs for this bug before submitting:
image

- strange background on the form:
image

**Comment by carkod** [comment]
@lyubomir-popov I pinged you several times in MM, about the decimal points, let's have a meeting.

**Comment by juanruitina** [comment]
UX+1

File: templates/download/intel-iei-tank-870.html [hunk 1]
```diff
@@ -28,66 +28,58 @@ <h1>Install Ubuntu Core<br /> on an Intel IEI TANK 870</h1>
...
-            <li><p>Run the following command, where <code>&lt;disk label&gt;</code> is the label of the second USB flash drive:</p>
-            <pre><code>xzcat /media/ubuntu/&lt;disk label&gt;/&lt;name of the image&gt;.img.xz | sudo dd of=/dev/&lt;target disk device&gt; bs=32M status=progress; sync</code></pre></li>
-            <li>Reboot the system and remove the flash drives when prompted. It will then boot from the internal memory where Ubuntu Core has been flashed.</li>
-          </ol>
-        </div>
-      </li>
-      {% include "download/shared/_first-boot-setup.html" %}
-      {% include "download/shared/_login.html" %}
-    </ol>
+    <div class="u-fixed-width">
```
Comment (mtruj013) [inline templates/download/intel-iei-tank-870.html:31]: We're 100% losing stuff in the constant rebases. I pushed these changes days ago 😅

File: templates/download/intel-nuc-desktop.html [hunk 2]
```diff
@@ -1,90 +1,103 @@
...
+        <h2 class="p-stepped-list__title">
           Download Ubuntu Desktop
-        </h3>
+        </h2>
         <div class="p-stepped-list__content">
           <p>Download the correct Ubuntu image for your device:</p>
-          <ul class="u-no-margin--left u-no-margin--top">
-            <li><a href="http://people.canonical.com/~platform/images/nuc/pc-dawson-xenial-amd64-m4-20180507-32.iso"> Dawson Canyon and June Canyon NUC &ndash; Ubuntu Desktop 16.04 LTS image</a>
+          <hr class="p-rule" />
+          <ul class="p-list--divided u-no-margin--left u-no-margin--top">
```
Comment (mtruj013) [inline templates/download/intel-nuc-desktop.html:43]:
```suggestion
          <ul class="p-list--divided">
```
These 2 aren't needed,  vanilla applies `margin-left: 0` to divided lists and `margin-top: 0` to uls natively

File: templates/download/intel-nuc-desktop.html [hunk 3]
```diff
@@ -1,90 +1,103 @@
...
-            <li>Start the NUC and push F10 to enter the boot menu.</li>
-            <li>Select the USB flash drive as a boot option.</li>
-            <li>The system will automatically execute the first stage of installation and prompt for an acknowledgement of a complete system recovery.</li>
-            <li>Follow the instructions and enter appropriate options for language, WiFi, location (timezone), and keyboard layout.</li>
-            <li>Pick a hostname, user account and password.</li>
-            <li>Wait for the configuration to finish. If you are connected to an active network, it will take several minutes to download and apply additional updates.</li>
-            <li>Ubuntu is installed. Use your account and password to log in.</li>
-          </ol>
+          <hr class="p-rule" />
+          <ol class="u-no-margin--left p-list--divided">
```
Comment (mtruj013) [inline templates/download/intel-nuc-desktop.html:66]:
```suggestion
          <ol class="p-list--divided">
```

File: templates/download/intel-nuc-desktop.html [hunk 4]
```diff
@@ -1,90 +1,103 @@
...
+            <li class="p-list__item">Insert the USB flash drive in the NUC.</li>
+            <li class="p-list__item">Start the NUC and push F10 to enter the boot menu.</li>
+            <li class="p-list__item">Select the USB flash drive as a boot option.</li>
+            <li class="p-list__item">The system will automatically execute the first stage of installation and prompt for an acknowledgement of a complete system recovery.</li>
+            <li class="p-list__item">Follow the instructions and enter appropriate options for language, WiFi, location (timezone), and keyboard layout.</li>
+            <li class="p-list__item">Pick a hostname, user account and password.</li>
+            <li class="p-list__item">Wait for the configuration to finish. If you are connected to an active network, it will take several minutes to download and apply additional updates.</li>
+            <li class="p-list__item">Ubuntu is installed. Use your account and password to log in.</li>
+          </ul>
```
Comment (mtruj013) [inline templates/download/intel-nuc-desktop.html:75]:
```suggestion
          </ol>
```

File: templates/download/intel-nuc-desktop.html [hunk 5]
```diff
@@ -1,90 +1,103 @@
...
+<script>
+  canonicalLatestNews.fetchLatestNews(
+    {
+      articlesContainerSelector: "#latest-articles",
+      articleTemplateSelector: "#article-template",
+      limit: "5",
+      groupId: "1666",
+    }
+  )
+</script>
```
Comment (mtruj013) [inline templates/download/intel-nuc-desktop.html:102]:
```suggestion
```
No need to add this, it's already included in the resources partial

File: templates/download/shared/_first-boot-setup.html [hunk 6]
```diff
@@ -8,11 +8,11 @@ <h2 class="p-stepped-list__title">First boot setup</h2>
       <li class="p-list__item">Press enter then select "Start" to begin configuring your network and an administrator account. Follow the instructions on the screen, you will be asked to configure your network and enter your Ubuntu SSO credentials.</li>
       <li class="p-list__item">
         At the end of the process, you will see your credentials to access your Ubuntu Core machine:
-        <p></p>
+        <br /><br />
```
Comment (mtruj013) [inline templates/download/shared/_first-boot-setup.html:11]: Same comment here as on the other pr, I believe it's best practice to use margin to add spacing here

**Review by mtruj013 — APPROVED** [review]
LGTM, with a couple of minor changes

File: templates/download/intel-iei-tank-870.html (hunk 1 above)
Comment (carkod) [reply templates/download/intel-iei-tank-870.html:31]: Sorry, let me rebase

File: templates/download/intel-nuc-desktop.html (hunk 3 above)
Comment (carkod) [reply templates/download/intel-nuc-desktop.html:66]: Is this not needed?

File: templates/download/shared/_first-boot-setup.html (hunk 6 above)
Comment (carkod) [reply templates/download/shared/_first-boot-setup.html]: Fixed with the rebase

**Pull request merged**